runtime/rasdump/javadump.cpp

```diff
@@ -48,6 +48,9 @@
 #include "omr.h"
 #include "omrutilbase.h"
 #include "j9version.h"
+#if defined(OPENJ9_BUILD)
```

Comment:
The required change to set OPENJ9_BUILD isn't added to the module.xml so I don't think it will ever be defined.